lib/src/over_react_test/common_component_util.dart

```diff
@@ -390,17 +390,26 @@ void testRequiredProps(BuilderOnlyUiFactory factory, dynamic childrenFactory(),
   var nullableProps = <String>[];
   var requiredProps = <String>[];
 
+  // Because Component2's propTypes do not stop the component from
+  // rendering with missing required props, an error may need to be caught
+  // on mount to keep the tests running.
+  void wrapInTryCatch(Function callback) {
```

Comment:
#nit This could use more specific typing
```suggestion
  void wrapInTryCatch(void callback()) {
```
(avoiding new function syntax to avoid Dart 1 regressions)